projects/scripts/adi_project_xilinx.tcl

```diff
@@ -13,12 +13,14 @@ if {[info exists ::env(ADI_IGNORE_VERSION_CHECK)]} {
 
 ## Define the ADI_USE_OOC_SYNTHESIS environment variable to enable out of context
 #  synthesis
-if {[info exists ::env(ADI_USE_OOC_SYNTHESIS)]} {
-  set ADI_USE_OOC_SYNTHESIS 1
-} elseif {![info exists ADI_USE_OOC_SYNTHESIS]} {
-  set ADI_USE_OOC_SYNTHESIS 0
+# if {[info exists ::env(ADI_USE_OOC_SYNTHESIS)]} {
+#   set ADI_USE_OOC_SYNTHESIS 1
+# } elseif {![info exists ADI_USE_OOC_SYNTHESIS]} {
+#   set ADI_USE_OOC_SYNTHESIS 0
+# }
 
-}
+## For Vivado 2020.2, set ADI_USE_OOC_SYNTHESIS to 1 in order to force Out Of Context mode
+  set ADI_USE_OOC_SYNTHESIS 1
```

Comment:
I think we should query here the value of the environment variable and set the mode based on that.
If the variable does not exists then set it to 1 by default.   

Something like the the ADI_MAX_OOC_JOBS implementation from below.